### PR TITLE
hide searchbar if user is not authenticated

### PR DESCRIPTION
**Pull request by wipascal**

### CODE REVIEW

File: bread/layout/skeleton.py [hunk 1]
```diff
@@ -39,7 +39,11 @@ def default_page_layout(menu, *content):
             ShellHeader(
                 hg.C("PLATFORMNAME"),
                 hg.C("COMPANYNAME"),
-                searchbar=hg.F(lambda c: c.get("SEARCHBAR", "")),
+                searchbar=hg.If(
+                    hg.C("request.user.is_authenticated"),
+                    hg.F(lambda c: c.get("SEARCHBAR", "")),
```
Comment (saemideluxe) [inline bread/layout/skeleton.py:44]: I think ```hg.C("SEARCHBAR")``` would work the same and be a bit shorter.

File: bread/layout/skeleton.py [hunk 2]
```diff
@@ -39,7 +39,11 @@ def default_page_layout(menu, *content):
             ShellHeader(
                 hg.C("PLATFORMNAME"),
                 hg.C("COMPANYNAME"),
-                searchbar=hg.F(lambda c: c.get("SEARCHBAR", "")),
+                searchbar=hg.If(
+                    hg.C("request.user.is_authenticated"),
+                    hg.C("SEARCHBAR", ""),
```
Comment (saemideluxe) [inline bread/layout/skeleton.py:44]: ```hg.C``` does only support a single argument... Did this not raise an exception? (If the value does not exists ```None``` is returned which is rendered as an empty string (i. e. not rendered).

Comment (saemideluxe) [reply bread/layout/skeleton.py:44]: Thinking about it: Adding support for a default value could actually make sense in some cases...

Comment (wipascal) [reply bread/layout/skeleton.py]: > `hg.C` does only support a single argument... Did this not raise an exception? (If the value does not exists `None` is returned which is rendered as an empty string (i. e. not rendered).

fixed it, thanks.

**Review by saemideluxe — APPROVED** [review]

**Pull request merged**